src/Nethermind/Nethermind.Merge.Plugin/GC/GCKeeper.cs

```diff
@@ -165,9 +167,7 @@ private async Task ScheduleGCInternal()
                     GCSettings.LargeObjectHeapCompactionMode = GCLargeObjectHeapCompactionMode.CompactOnce;
                 }
 
-                System.GC.Collect((int)generation, mode, blocking: true, compacting: compacting > 0);
-
-                MallocHelper.Instance.MallocTrim((uint)1.MiB());
+                GCScheduler.Instance.GCCollect((int)generation, mode, blocking: true, compacting: compacting > 0);
```

Comment:
should lines [165-168](https://github.com/NethermindEth/nethermind/pull/7432/files#diff-79d27af20a7736bf17e78b4578c54be208f0753194a14cfa53557ae2f9a6153fR165-R168) go into this method in GCScheduler?